type-definitions/most.d.ts

```diff
@@ -21,12 +21,59 @@ declare interface Iterable<A> {}
 
 declare type CreateGenerator<A> = (...args: Array<any>) => Generator<A|Promise<A>, any, any>;
 
+export interface Sink<A> {
+  event(time: number, value: A): void;
+  end(time: number, value: A): void;
```

Comment:
I think we'll need to make the event and end types different.  For example, `from(arrayOfNumbers)` would require a sink with an event type `number`, and end type `void` (or undefined, or whatever TS uses).  Hmmm, either that or the end value needs to be optional, i.e. `value: ?A` or however TS does that.

What do you think is best?